common/src/main/java/com/discordsrv/common/config/main/AvatarProviderConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.discordsrv.common.config.main;
+
+import com.discordsrv.common.config.annotation.Untranslated;
+import org.spongepowered.configurate.objectmapping.ConfigSerializable;
+import org.spongepowered.configurate.objectmapping.meta.Comment;
+
+@ConfigSerializable
+public class AvatarProviderConfig {
+
+    @Comment("Whether to let DiscordSRV decide an appropriate avatar URL\n" +
+            "This will result in appropriate head renders being provided for Bedrock and Offline Mode players.")
```

Comment:
```suggestion
    @Comment("Whether to let DiscordSRV decide an appropriate avatar URL automatically\n" +
            "This will result in appropriate head renders being provided for Bedrock players (when using Floodgate) and Offline Mode players (via username).")
```